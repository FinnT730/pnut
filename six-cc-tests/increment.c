int main() {
  int a;
  int b;
  int x;
  int y;
  int z;

  x = 3;
  y = x++;
  x = 3;
  z = ++x;

  a = 0;
  b = 0;
  while (a < 10) {
    b += a++;
  }
  printf("a: %d, b: %d, x: %d, y: %d, z: %d\n", a, b, x, y, z);
}
