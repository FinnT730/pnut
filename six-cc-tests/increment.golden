a: 10, b: 45, x: 4, y: 3, z: 4
